frontend/packages/ceph-storage-plugin/src/utils/dashboard.ts

```diff
@@ -0,0 +1,11 @@
+import * as _ from 'lodash';
+import { PrometheusResponse } from '@console/internal/components/graphs';
+
+export const getResiliencyProgress = (results: PrometheusResponse): number => {
+  const progress = _.get(results, 'data.result[0].value[1]');
```

Comment:
nit: can you make this as a selector? This is being used at multiple [places in both dashboards.

Reply:
can be done in separate PR, which will change all occurrences of a single stat values.